### PR TITLE
New warning row for non-existent entities

### PR DESCRIPTION
**Pull request by iantrich**

### CODE REVIEW

File: src/panels/lovelace/entity-rows/hui-error-entity-row.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  public entity?: string;
+
+  static get properties() {
+    return {
+      entity: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    if (!this.entity) {
```
Comment (balloob) [inline src/panels/lovelace/entity-rows/hui-error-entity-row.ts]: I think that we should always render this element, even if no entity set. Make it `${this.enttiy || ''}` below and it's fine

**Review by balloob — APPROVED** [review]
ok to merge when comment addressed.

File: src/panels/lovelace/entity-rows/hui-error-entity-row.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <div>
+        Entity not available: ${this.entity}
+      </div>
+    `;
+  }
+
+  private renderStyle(): TemplateResult {
+    return html`
+      <style>
+        div {
```
Comment (balloob) [inline src/panels/lovelace/entity-rows/hui-error-entity-row.ts]: Oh, one more comment. Skip the whole div. Remove it from the template and update CSS to style  `:host`

```css
:host {
  display: block;
  background-color: yellow;
  padding: 8px;
}
```

btw do we know why we use `flex: 1`?

Comment (iantrich) [reply src/panels/lovelace/entity-rows/hui-error-entity-row.ts]: I do not

File: src/panels/lovelace/entity-rows/hui-error-entity-row.ts (hunk 1 above)
Comment (balloob) [reply src/panels/lovelace/entity-rows/hui-error-entity-row.ts]: I also indirectly meant that we should remove this if, or else it doesn't make sense to default below.

File: src/panels/lovelace/entity-rows/hui-error-entity-row.ts (hunk 2 above)
Comment (balloob) [reply src/panels/lovelace/entity-rows/hui-error-entity-row.ts]: I can see that we only work with the best engineers here 👍 

Switch to `:host` was good, I removed the `<div>` as it is no longer needed.

**Review by balloob — APPROVED** [review]

File: src/panels/lovelace/entity-rows/hui-error-entity-row.ts (hunk 2 above)
Comment (iantrich) [reply src/panels/lovelace/entity-rows/hui-error-entity-row.ts]: Hey, you brought on a C++ dev that start front-end stuff three months ago 🤣
I am very much the "well it works" crowd right now, but am trying to learn the why as well.

**Pull request merged**